### PR TITLE
Update hololens-calibration.md

### PR DESCRIPTION
**Pull request by kevincol**

### CODE REVIEW

**Comment by illfated** [comment]
Ref. #5113 (different wording in the PR, but still empty description)

File: devices/hololens/hololens-calibration.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ You can also disable the calibration prompt by following these steps:
 1. Turn off **When a new person uses this HoloLens, automatically ask to run eye calibration**.
 
 > [!IMPORTANT]
-> Please understand that this setting may adversely affect hologram rendering quality and comfort.
+> Please understand that this setting may adversely affect hologram rendering quality and comfort.  If there is an immersive application that is using eye tracking, for instance text scrolling, then that feature will no longer work.
```
Comment (Teresa-Motiv) [inline devices/hololens/hololens-calibration.md]: Edited text of the proposed sentence:
 When you turn off this setting, features that depend on eye tracking (such as text scrolling) no longer work in immersive applications.

**Comment by e0i** [comment]
@kevincol Would you be able to follow up with revision suggestions provided by the document author, *Teresa-Motiv*? We would like to proceed with this PR. Thanks.

**Comment by illfated** [comment]
Please also consider which one of the Pull Requests (#5113 , #5114 ) to keep, because they will be in conflict with each other, targeting the same file in the same branch.

**Comment by Dansimp** [comment]
applied teresa's change, plus one of my own (removed 'please understand').

**Comment by illfated** [comment]
Looks good to me, except for maybe a cosmetic grammar detail. I would prefer to see something like
> features depending on eye tracking

instead of

> features that depend on eye tracking

Then again, with English not being my first language, I might be more used to reading a different kind of literature, making my suggestion a moot point.

**Comment by Teresa-Motiv** [comment]
the "features that depend on" structure is consistent with Microsoft style

**Comment by e0i** [comment]
@Teresa-Motiv @Dansimp Would you be able to merge the PR given the revision suggestions have been applied? Thanks.

**Comment by e0i** [comment]
@Dansimp The PR looks like ready to be merged.

**Comment by Teresa-Motiv** [comment]
@Dansimp, can you merge this? I don't appear to have permissions to do so.

Thanks.

**Pull request merged**